troposphere/themes/jetstream_theme/scss/theme.scss

```diff
@@ -0,0 +1,72 @@
+
+// Basics of the navbar
+$navbar-default-bg: white;
+$brand-primary: #E5004A;
+$brand-secondary: #00AAD5;
+$logo-url: '../../images/mini_logo.png';
+$application-name: '';
+$application-label: 'DEMO';
+
+// Navbar links
+$navbar-default-link-color: darken(lightgrey, 10%);
+$navbar-default-link-hover-color: black;
+$navbar-default-link-active-color: black;
+$navbar-default-link-active-bg: darken($navbar-default-bg, 6.5%);
+
+// Navbar brand label
+$navbar-default-brand-color: $brand-primary;
+$navbar-default-brand-hover-color: $navbar-default-brand-color;
+
+// Navbar toggle
+$navbar-default-toggle-hover-bg: $navbar-default-link-active-bg;
+$navbar-default-toggle-icon-bar-bg: #fff;
+$navbar-default-toggle-border-color: $navbar-default-link-active-bg;
+
+// Secondary Navbar
+$secondary-nav-border: $brand-primary;
+
+// UI Elements
+$link-color: $brand-secondary;
+$tag-color: $brand-primary;
+
+
+
+@import "nav";
+@import "ui";
+
+#dashboard-view {
```

Comment:
Will break this out into it's own partial called _content.scss or something next pass.